.github/stale.yml

```diff
@@ -11,7 +11,14 @@ staleLabel: wontfix
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  recent activity. It will be closed if no further activity occurs.
+  Please note: We are just a few people who contribute to a shared project,
+  it's impossible for us to fix every bug with such limited resources.
```

Comment:
I think that in here there needs to be something about people trying to submit PRs.  Maybe
"If you want to investigate and try to help solve this yourself, we encourage and will review all pull requests from new contributors"

Reply:
Got it, how about that?